updates.jenkins.io.tf

```diff
@@ -66,14 +66,14 @@ output "updates_jenkins_io_redirections_fileshare_name" {
 
 # Redis database
 resource "azurerm_redis_cache" "updates_jenkins_io" {
-  name                = "updates-jenkins-io"
-  location            = azurerm_resource_group.updates_jenkins_io.location
-  resource_group_name = azurerm_resource_group.updates_jenkins_io.name
-  capacity            = 1
-  family              = "C"        # Basic/Standard SKU family
-  sku_name            = "Standard" # A replicated cache in a two node Primary/Secondary configuration managed by Microsoft, with a high availability SLA.
-  enable_non_ssl_port = true
-  minimum_tls_version = "1.2"
+  name                 = "updates-jenkins-io"
+  location             = azurerm_resource_group.updates_jenkins_io.location
+  resource_group_name  = azurerm_resource_group.updates_jenkins_io.name
+  capacity             = 1
+  family               = "C"        # Basic/Standard SKU family
+  sku_name             = "Standard" # A replicated cache in a two node Primary/Secondary configuration managed by Microsoft, with a high availability SLA.
+  non_ssl_port_enabled = true
```

Comment:
weird, this change shows the following plan:

```
# azurerm_redis_cache.updates_jenkins_io will be updated in-place
  ~ resource "azurerm_redis_cache" "updates_jenkins_io" {
      ~ enable_non_ssl_port           = true -> false
```

Gotta check with the upstream provider repo: until then no merge!

Reply:
Ref. https://github.com/hashicorp/terraform-provider-azurerm/issues/26943